File: src/minigames/paintball/java/work/lclpnet/ap2/game/paintball/util/Paintable.java
package work.lclpnet.ap2.game.paintball.util;

import net.minecraft.world.level.block.Block;
import work.lclpnet.ap2.api.game.team.DyeTeamKey;

public interface Paintable {

    Block blockFor(DyeTeamKey team);
}

File: src/minigames/paintball/src/main/kotlin/work/lclpnet/ap2/game/paintball/util/Paintable.kt
package work.lclpnet.ap2.game.paintball.util

import net.minecraft.world.level.block.Block
import work.lclpnet.ap2.api.game.team.DyeTeamKey

fun interface Paintable {
    fun blockFor(team: DyeTeamKey): Block
}
